modules/virt-tested-maximums.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/virt-supported-limits.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-tested-maximums_{context}"]
+= Tested maximums for {VirtProductName}
+
+The following limits apply to a large-scale {VirtProductName} 4.x environment. They are based on a single cluster of the largest possible size. When you plan an environment, remember that multiple smaller clusters might be the best option for your use case.
+
+[id="vm-maximums_{context}"]
+== Virtual machine maximums
+
+The following maximums apply to virtual machines (VMs) running on {VirtProductName}. These values are subject to the limits specified in link:https://access.redhat.com/articles/rhel-kvm-limits[Virtualization limits for Red Hat Enterprise Linux with KVM].
```

Comment:
Per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces), there should be a non breaking space in between Red and Hat:
```suggestion
The following maximums apply to virtual machines (VMs) running on {VirtProductName}. These values are subject to the limits specified in link:https://access.redhat.com/articles/rhel-kvm-limits[Virtualization limits for Red{nbsp}Hat Enterprise Linux with KVM].
```